be/src/storage/rowset/bitmap_index_writer.cpp

```diff
@@ -173,6 +218,13 @@ class BitmapIndexWriterImpl : public BitmapIndexWriter {
     uint64_t size() const override {
         uint64_t size = 0;
         size += _null_bitmap.getSizeInBytes(false);
+        for (RoaringWrapper* roaring_wrapper : _size_changed_roaring_vec) {
+            uint64_t new_size = roaring_wrapper->roaring_size();
+            _reverted_index_size += (new_size - roaring_wrapper->old_size());
```

Comment:
1. new_size and old_size is hard to understand for me.

Reply:
If the `_element_count` is larger than `roaring_get_size_fastpath_threshold`, each time we will use the `roaring_wrapper->roaring_size();` to obtain the exact size of roaring bitmap.
The bitmap index's `size()` is called every time we want to append a new chunk to the segment, if the segment's size is larger than threshold, we need to first flush the segment.
So here the old_size represents the size of  roaring bitmap in the last round of chunk appended. 
The new_size represents the size of roaring bitmap in the current round of chunk appended.
The gap between them is the increasing size, and we add the diff to the _reverted_index_size.